SwapnilPatle.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Swapnil Patle",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Front End Developer.Learner,Youtuber",
+  "avatar": "https://github.com/settings/profile",
```

Comment:
Change this also accordingly to your GitHub profile URL and just add `.png` at the end. Eg `https://github.com/AbbeyIT.png`